en/doc_directory.md

```diff
@@ -0,0 +1,38 @@
+# Document Directory
+
+SCRF's Document Directory is a comprehensive guide of all SCRF documentation.
+
+## Awards, Bounties, Grants
+
+* [Research Summary Grants](en/grants_general_contributors.md)
+* [Pan-African Microgrants](en/grants_pan_african.md)
+* [Contributor Recognition and Rewards Initiative](en/grants_contributor_rewards.md)
+* [Impact Award](en/grants_impact_award.md)
+
+## Events
+
+* [Weekly Community Calls](en/content_community_calls)
+
+## Contributor Resources
+
+* [How to Get Compensated for Work](en/content_how_to_get_compensated_for_work.md)
+* [Research Improvement Process](en/process_research_improvement_proposal.md)
+* [Research Improvement Proposal](en/project_research_improvement_proposal.md)
```

Comment:
Didn't one or more of these file names change recently?

Reply:
The research improvement proposal is still active. The PR is not yet approved to change the file.